k8s/demo/cluster-00-overlay/em/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ bases:
 - ../../../namespaces/em/em-icp/em-icp.yaml
 - ../../../namespaces/em/em-hrs-ingestor/em-hrs-ingestor.yaml
 patchesStrategicMerge:
-- ../../../namespaces/em/em-hrs-ingestor/schedule-cluster-0.yaml
```

Comment:
are schedule-cluster-0.yaml and schedule-cluster-1.yaml to be  removed?

Reply:
they are not being removed per se, they are being replaced by a different config:
- ../../../namespaces/em/em-hrs-ingestor/demo.yaml